cmd/maintenance-manager/main.go

```diff
@@ -132,20 +138,30 @@ func main() {
 		os.Exit(1)
 	}
 
+	k8sInterface, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		setupLog.Error(err, "unable to create kubernetes interface")
+		os.Exit(1)
+	}
+
+	mgrClient := mgr.GetClient()
```

Comment:
Do we need to explicitly share the client? I think mgr.GetClient() will always return the same instance from the manager

Reply:
we dont, its just shorter to specify than mgr.GetClient() i can change if thats your preference LMK

Reply:
No It is ok to keep it like this, I just want to check that I'm not missing anything.